routers/api/v1/repo/language.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package repo
+
+import (
+	"bytes"
+	"net/http"
+	"strconv"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/log"
+)
+
+type languageResponse []*models.LanguageStat
+
+func (l languageResponse) MarshalJSON() ([]byte, error) {
```

Comment:
pleace move this convertion into modules/convert

Reply:
It's not really conversation

Reply:
@lunny what do you think?
at the moment I think it is a convert and lafriks dont

so the 3rd will decide ;)

Reply:
In convert module there are just convertation between types. In this case it is specific internal type serialziation/encoding to json